src/pages/SearchDetails/components/SearchItem/SearchItem.tsx

```diff
@@ -55,42 +71,52 @@ export default function SearchItem({ shop, addCard }: Props) {
           </div>
 
           <div className={`${styles.info__state} ${styles.state}`}>
-            {isMobile ? (
+            {isMobile && (
               <>
                 <Star />
-                <div className={styles.state__time}>0.0</div>
+                <div className={styles.state__rating}>{rate}</div>
                 |
-                <div className={styles.state__open}>{openNow ? '영업중' : '영업 종료'}</div>
               </>
-            ) : (
+            )}
+
+            <div
+              className={cn({
+                [styles.state__operating]: true,
+                [styles['state__operating--open']]: !!openNow,
+              })}
+            >
+              {openNow ? '영업중' : '영업 종료'}
+            </div>
+
+            {!isMobile && (
               <>
-                <div className={styles.state__open}>{openNow ? '영업중' : '영업 종료'}</div>
                 |
-                <div className={styles.state__time}>Operating Time</div>
+                <div className={styles.state__time}>
+                  {openNow ? openTime : closeTime}
+                </div>
               </>
             )}
           </div>
-
-          {!isMobile && (
+          {/* {!isMobile && (
             <div className={`${styles.info__call} ${styles.call}`}>
               <PhoneIcon className={styles.call__icon} />
-              <div className={styles.call__number}>Phone Number</div>
+              <div className={styles.call__number}>{safePhoneNumber}</div>
             </div>
-          )}
+          )} */}
         </div>
-        <div className={`${styles.box__pictures} ${styles.pictures}`}>
-          <picture className={styles.pictures__picture}>
-            <img className={styles.pictures__image} alt="가게 이미지" src={photoToken ?? defaultImage} />
-          </picture>
-          {/* <picture className={styles.pictures__picture}>
-            <source srcSet={defaultImage} />
-            <img className={styles.pictures__image} alt="가게 이미지" />
-          </picture>
-          <picture className={styles.pictures__picture}>
-            <source srcSet={defaultImage} />
-            <img className={styles.pictures__image} alt="가게 이미지" />
-          </picture> */}
+
+        <div className={`${styles.box__images} ${styles.images}`}>
+          <img
+            className={styles.images__image}
+            alt="가게 이미지"
+            src={photoToken ?? defaultImage}
+            onError={(e) => {
+              const target = e.target as HTMLImageElement;
+              target.src = defaultImage;
+            }}
```

Comment:
type assertion은 왠만해서는 쓸일이 없어요~! 이 경우에도 아래와 같이 하는 것이 더 좋을 것 같네요~!

```suggestion
            onError={(e) => {
              if(e.target instanceof HTMLImageElement) {
                e.target.src = defaultImage;
            }}
```

Reply:
와~ 몰랐던 방법이에요!